contracts/DelayEvents.sol

```diff
@@ -18,7 +18,12 @@ contract DelayEvents {
         uint256[] CliffDelays,
         uint256[] FinishDelays
     );
-    event BoughtBackTokens(
+    event TokenRedemptionApproval(
+        address indexed token,
+        address indexed user,
+        bool status
+    );
+    event RedeemedTokens(
```

Comment:
What do you think about adding the "Owner" field to the RedeemedTokens event?

Reply:
I don't see a reason. only if we add address to the redemption, then we want to know who get the tokens